GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -716,9 +729,23 @@ private static String dateCellValue(String value) {
         return ExportImportUtils.formatDateResponse(value);
     }
 
-    private static String photoCellValue(String value, String imagePrefix) {
-        return ExportImportUtils.formatImage(imagePrefix, value);
-    }
+    private static List<String> mediaCellValues(String value, boolean useQuestionId, String imagePrefix) {
+        List<String> cells = new ArrayList<>();
+        if (value.startsWith("{")) { //JSON, possibly geotagged
```

Comment:
I didn't execute but this seems like it will skip the non-JSON formatted responses altogether... right? Perhaps we don't even need this if block. Only check later if there is location data and create the relevant cells.